code/modules/paperwork/folders.dm

```diff
@@ -55,6 +55,8 @@
 		dat += "<A href='?src=\ref[src];remove=\ref[Ph]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Ph]'>Rename</A> - <A href='?src=\ref[src];look=\ref[Ph]'>[Ph.name]</A><BR>"
 	for(var/obj/item/weapon/paper_bundle/Pb in src)
 		dat += "<A href='?src=\ref[src];remove=\ref[Pb]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Pb]'>Rename</A> - <A href='?src=\ref[src];browse=\ref[Pb]'>[Pb.name]</A><BR>"
+	for(var/obj/item/weapon/sample/print/Pf in src)
+		dat += "<A href='?src=\ref[src];remove=\ref[Pf]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Pf]'>Rename</A> - <A>[Pf.name]</A><BR>"
```

Comment:
Why did you add a button to rename it, if you cant rename it.
Remove the rename button here.

Reply:
The original plan was to keep the cards in-line with the things already in the folder. They had rename buttons, so to keep the fingerprint card from being slid over a full word, I'd keep the button even though the cards named themselves. 

Then I actually started sampling things. Apparently the names can get quite long.

Taking out the rename button is the better play, visually.